### PR TITLE
Add ITT Mentor Services to service docs

### PR DESCRIPTION
**Pull request by JamieCleare2525**

### CODE REVIEW

**Review by saliceti — APPROVED** [review]

File: config/services.yml [hunk 1]
```diff
@@ -143,3 +143,8 @@
   csproj_path: 'GetIntoTeachingApi/GetIntoTeachingApi.csproj'
   docsets:
     - path: 'docs'
+- name: "ITT Mentor Services"
+  repo_name: 'DFE-Digital/itt-mentor-services'
+  language: 'ruby'
+  docsets:
+    - path: 'docs'
```
Comment (ollietreend) [inline config/services.yml:150]: Our ADRs are in a separate directory, so I'm wondering if we want to include that path too?

Similar to what the 'Register trainee teachers' service does:

https://github.com/DFE-Digital/teacher-services-tech-docs/blob/9ceaa7a3bf032fedb4b8db4bad930dd330276d74/config/services.yml#L15-L20

**Review by ollietreend — COMMENTED** [review]
This looks good – just one comment

File: config/services.yml (hunk 1 above)
Comment (JamieCleare2525) [reply config/services.yml]: Resolved in [4635543](https://github.com/DFE-Digital/teacher-services-tech-docs/pull/49/commits/4635543aec47b8765eafff885a7e3763482a33a8)

**Review by ollietreend — APPROVED** [review]
Looks good to me @JamieCleare2525 👍🏻

Could we squash these commits in to one before merging please? 🙏🏻

**Pull request merged**